utils/metrics.py

```diff
@@ -0,0 +1,19 @@
+from prometheus_client import Gauge
+from prometheus_client import Counter
+from prometheus_client import Histogram
+
+
+run_time = Gauge(name='qontract_reconcile_last_run_seconds',
+                 documentation='Last run duration in seconds',
+                 labelnames=['integration', 'shards', 'shard_id'])
+
+run_status = Counter(name='qontract_reconcile_run_status',
+                     documentation='Status of the runs',
+                     labelnames=['integration', 'status',
+                                 'shards', 'shard_id'])
+
+func_runtime = Histogram(name='qontract_reconcile_function_runtime',
```

Comment:
would be `qontract_reconcile_function_elapsed_seconds_since_bundle_commit` too long? 😂

Reply:
not for me. hopefully also not for prometheus.

Reply:
It doesn't look like it : https://prometheus.io/docs/concepts/data_model/